client/src/modules/dashboard/index.jsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import ShoppingList from 'modules/shopping-list';
+
 const Dashboard = ({
   match: {
     params: { id }
   }
-}) => <div>Dashboard</div>;
+}) => (
+  <div>
+    <ShoppingList />
```

Comment:
does full Shopping List should  be displayed in a dashboard? Shoping list module is better place to put it in..

Reply:
It's only a temporary solution. In the next task, you will move it to another view.